### DISCUSSION TITLE
请问可以加入播放器对flv的支持吗？

### DISCUSSION THREAD

**Discussion by Yuukipix** [AlistGo/alist#292] [Ideas]
请问可以加入播放器对flv的支持吗？

**Comment by xhofe** [comment]
https://alist.xhofe.top/%E9%98%BF%E9%87%8C%E4%BA%91%E7%9B%98/%E8%A7%86%E9%A2%91/test.flv

除阿里其他存储下个版本加入

**Reply by Yuukipix** [reply]
感谢！
